packages/hooks/src/useEventLogger.ts

```diff
@@ -31,28 +31,30 @@ export const useEventLoggerStore = create<EventLoggerStore>(
   }),
 )
 
+const logEventAtom = (store: EventLoggerStore) => store.logEvent
+const setEventReducersAtom = (store: EventLoggerStore) => store.setEventReducers
+const eventReducersAtom = (store: EventLoggerStore) => store.eventReducers
```

Comment:
@alexanderchan can we call them selectors since that is what zustand calls them and if we adopt jotai we can call those atoms to keep the nomenclature separate?